draft-ietf-ppm-dap.md

```diff
@@ -1461,6 +1461,16 @@ The Helper responds to the Leader with HTTP status code 201 Created and a body
 consisting of the `AggregationJobResp`, with media type
 "application/dap-aggregation-job-resp".
 
+Changing an aggregation job's parameters is illegal, so further requests to
+`PUT /tasks/{tasks}/aggregation_jobs/{aggregation-job-id}` for the same
+`aggregation-job-id` but with a different `AggregationJobInitReq` in the body
+MUST fail with an HTTP client error status code.
```

Comment:
> ... MUST fail with an HTTP client error status code.

I think we should be more prescriptive than this. It seems like a situation where the Leader would want to take a different action based on the feedback from the Helper.

Reply:
I don't agree. What this paragraph says is that changing an aggregation job's parameters is illegal, undefined, outside of what this protocol discusses. So it does not make sense to design in-band mechanisms to recover from this scenario. If this ever happens, then one of the protocol participants has a bug and the way to recover is to make software changes.